platform/api/gcloud/image.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2016 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package gcloud
+
+import (
+	"fmt"
+	"strings"
+
+	"google.golang.org/api/compute/v1"
+)
+
+// CreateImage creates an image on GCE and and wait for completion. If
+// overwrite is true, an existing image will be overwritten if it exists.
+func (a *API) CreateImage(name, source string, overwrite bool) error {
```

Comment:
In order for this to be useful for plume it is going to need to accept a full compute.Image because there are a number of options that need to be set like family and whatnot. That's what my aborted attempt at the api refactor was doing.
￼